acceptance/src/main/java/io/github/tcdl/msb/acceptance/payload/Body.java

```diff
@@ -0,0 +1,14 @@
+package io.github.tcdl.msb.acceptance.payload;
+
+public class Body {
+
+    private String body;
```

Comment:
Can we make the field name different to make it less confusing? To avoid such code: new Body().setBody(...)